### PR TITLE
ffmpeg : bump

### PR DESCRIPTION
**Pull request by extrowerk**
We should consider rename all the recipes for 3.x to ffmpeg3.
We need to define CONFLICTS against the ffmpeg3 devel packages.

### CODE REVIEW

**Comment by extrowerk** [comment]
Anybody up to the task to review this?
@alaviss @fbrosson @waddlesplash

File: media-video/ffmpeg/ffmpeg-4.0.1.recipe [hunk 1]
```diff
@@ -0,0 +1,239 @@
+SUMMARY="Audio and video recording, conversion, and streaming library"
+DESCRIPTION="FFmpeg is a complete, cross-platform solution to record, convert \
+and stream audio and video. It includes libavcodec - the leading audio/video \
+codec library."
+HOMEPAGE="https://ffmpeg.org/"
+COPYRIGHT="2000-2003 Fabrice Bellard
+	2003-2018 the FFmpeg developers"
+LICENSE="GNU LGPL v2.1
```
Comment (waddlesplash) [inline media-video/ffmpeg/ffmpeg-4.0.1.recipe]: v3

File: media-video/ffmpeg/ffmpeg-4.0.1.recipe [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+libswscaleVersion="5.1.100"
+libswresampleVersion="3.1.100"
+libpostprocVersion="55.1.100"
+portVersionCompat="$portVersion compat >= 4"
+for i in $ffmpegLibs; do
+	eval "${i}VersionCompat=\"\$${i}Version compat >= \${${i}Version%%.*}\""
+done
+
+PROVIDES="
+	ffmpeg4$secondaryArchSuffix = $portVersionCompat
```
Comment (fbrosson) [inline media-video/ffmpeg/ffmpeg-4.0.1.recipe]: You should **_either_** rename the recipe to `ffmpeg4-4.0.1.recipe` **_or_** `sed s/ffmpeg4/ffmpeg/` `PROVIDES{,_avdevice,_devel,_tools}`.

BTW, I'd recommend removing `ffmpeg-4.0.recipe` (and `patches/ffmpeg-4.0.patchset`.)

**Comment by waddlesplash** [comment]
@fbrosson, I've taken this one up offline, and have an almost-working pseudo-GCC2 adaptation of it. If you're around -- find me on IRC?

**Comment by waddlesplash** [comment]
Superseded by #2758.

**Pull request closed without merging**